tests/integration/suite/test_system_project.py

```diff
@@ -53,8 +53,9 @@ def test_system_namespaces_assigned(admin_cc):
     # will be present in current system_namespaces, removing it
     if loggingNamespace in system_namespaces_names:
         system_namespaces_names.remove(loggingNamespace)
-
-    assert system_namespaces_names == initial_system_namespaces
+    print(initial_system_namespaces)
```

Comment:
Should drop the print statement, the assert can be extended to include a string with the contents of the lists.